src/main/java/org/embulk/output/elasticsearch/ElasticsearchOutputPluginDelegate.java

```diff
@@ -236,7 +235,7 @@ public JacksonServiceRequestMapper buildServiceRequestMapper(PluginTask task)
         // Traditional constructor is used here for compatibility.
         final ConfigSource configSource = Exec.newConfigSource();
         configSource.set("format", "%Y-%m-%dT%H:%M:%S.%3N%z");
-        configSource.set("timezone", DateTimeZone.forID(task.getTimeZone()));
+        configSource.set("timezone", task.getTimeZone());
         TimestampFormatter formatter = TimestampFormatter.of(
```

Comment:
It could be better to use the other creator method of `TimestampFormatter`, instead of keep using it. We no longer need to create `ConfigSource` with it.
https://github.com/embulk/embulk/blob/v0.10.5/embulk-core/src/main/java/org/embulk/spi/time/TimestampFormatter.java#L41

Or, we can use `java.time.formatter.DateTimeFormatter` instead of Embulk's `TimestampFormatter`. I think it would be the choice we finally should choose, because this `TimestampFormatter` would be finally deprecated. But doing it in a later PR is fine.

JFYI, if we have to keep the Ruby-style timestamp format, this plugin-side library `embulk-util-timestamp` would be the choice instead of Embulk's `TimestampFormatter`.
* https://search.maven.org/artifact/org.embulk/embulk-util-timestamp
* https://dev.embulk.org/embulk-util-timestamp/0.2.0/javadoc/org/embulk/util/timestamp/TimestampFormatter.html

Reply:
Yes, that makes sense to use another creator method. I'll make changes.
 
In order to change `TimestampFormatter` type to anything else we need to look into alternatives for current  `JacksonAllInObjectScope`. The latest tag is using `org.embulk.spi.time.TimestampFormatter`. see
https://github.com/embulk/embulk-base-restclient/blob/v0.7.0/src/main/java/org/embulk/base/restclient/jackson/scope/JacksonAllInObjectScope.java
Regarding that, it looks like much more broader changes if we want to rid of `org.embulk.spi.time.TimestampFormatter`.

Reply:
Ah, got it. There has been a dependency there... I was unaware, forgotten. Okay, let's do that in another broader action. Thanks!